core/src/main/java/org/ehcache/core/events/StoreEventSink.java

```diff
@@ -27,7 +29,7 @@
    * @param key removed key
    * @param value removed value
    */
-  void removed(K key, V value);
+  void removed(K key, Invoker<V> value);
```

Comment:
This is necessary to not force deserialisation if no store event listeners are registered. Cache event listeners will anyway get the raw value, not wrapped.